.github/workflows/ci.yml

```diff
@@ -117,3 +117,15 @@ jobs:
           yarn --frozen-lockfile
           /usr/bin/Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
           yarn run test
+  verify_indexing:
```

Comment:
nit: I don't think we should use `verify` when we actually don't check the result's correctness. Maybe something like `indexing_sanity_check` will be closer to what it does?